brave/brave-core
# feat(wallet): Portfolio Customization Settings

## Description 
`Portfolio Customization` settings in the `Wallet More Popup`
- Moves `Hide balances` into the `Wallet More Popup`
- Moves `Show graph` into the `Wallet More Popup`


Resolves <https://github.com/brave/brave-browser/issues/29868>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Portfolio` page and open the `More Menu` in the header
2. Click on the `Show graph` option and test that the `Price graph` hides or un-hides
3. Refresh the page, the selection should be persisted.
4. Click on the `Hide balances` option and test that the `Balances` hide or un-hides
5. Refresh the page, the selection should be persisted.
6. Open the `Panel` and click on the `More Menu` in the header
7. You should not see any `Portfolio Customization` settings

https://user-images.githubusercontent.com/40611140/233472791-cb8b653c-7950-48d0-9885-98e602521441.mov
